conf/evolutions/default/8.sql

```diff
@@ -0,0 +1,15 @@
+# --- !Ups
+
+CREATE TABLE edition_issues_publication_history (
```

Comment:
`publication_history` might be a simpler name for this table

Reply:
if issue is the only entity that can be published